pallets/amm/src/tests.rs

```diff
@@ -283,6 +283,149 @@ fn remove_zero_liquidity_should_not_work() {
 	});
 }
 
+#[test]
+fn remove_liquidity_should_not_work() {
+	new_test_ext().execute_with(|| {
+		let user = ALICE;
+		let asset_a = HDX;
+		let asset_b = DOT;
+
+		assert_ok!(AMM::create_pool(
+			Origin::signed(user),
+			asset_a,
+			asset_b,
+			100_000_000,
+			Price::from(10_000)
+		));
+
+		let asset_pair = AssetPair {
+			asset_in: asset_a,
+			asset_out: asset_b,
+		};
+		assert_eq!(AMM::exists(asset_pair), true);
```

Comment:
you can just use assert.

```suggestion
		assert!(AMM::exists(asset_pair));
```